pkg/bindings/images/build.go

```diff
@@ -16,6 +16,8 @@ import (
 	"strconv"
 	"strings"
 
+	util2 "github.com/containers/podman/v5/pkg/bindings/internal/util"
+
```

Comment:
Any reason this is separated from the other imports?

Reply:
This is the [goimport tool](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) that sort/move the imports.